docs/src/content/docs/fr/components/asides.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Encarts
+description: Apprenez à utiliser les encarts dans Starlight pour afficher des informations secondaires à côté du contenu principal d'une page.
+---
+
+import { Aside } from '@astrojs/starlight/components';
+
+Pour afficher des informations secondaires à côté du contenu principal d'une page, utilisez le composant `<Aside>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Aside slot="aperçu">
+Incluez des informations supplémentaires non essentielles dans un `<Aside>`.
+</Aside>
+
+</Preview>
+
+## Import
+
+```tsx
+import { Aside } from '@astrojs/starlight/components';
+```
+
+## Utilisation
+
+Affichez un encart à l'aide du composant `<Aside>`.
```

Comment:
```suggestion
Affichez un encart (également connus sous le nom de « admonition » ou « aside » en anglais) en utilisant le composant `<Aside>`.
```

Looks like a part of the sentence was skipped, I grabbed it from the previous French version.

Reply:
Yes I deleted it because I've never heard of “admonition”, I hesitated to leave “aside” though.